### ISSUE TITLE
More `vello` graphics primitives

### ISSUE THREAD

**Issue by nixon-voxell** [nixon-voxell/bevy_motiongfx#31]
Creating animations typically requires more primitives to work with.

A good reference point of what we are lacking at the moment is the [motion canvas API docs](https://motioncanvas.io/api/2d/components).

Personally below are some of the must have components for creating a motion graphics video:

- [x] Rect
- [x] Circle
- [x] Bezier Curves
- [ ] SVG [in progress]
- [ ] Text (inc. text, code, math symbols) [in progress]
- [ ] Arrow/Ray
- [ ] Video
- [ ] Audio
- [ ] Custom Shader

**Comment by AbstractiveNord** [comment]
Up priority for Video/Audio integration. May be worth to have integration with [touying](https://github.com/touying-typ/touying).

**Comment by musjj** [comment]
Would love to see polygons too, but I guess this should be handled upstream (https://github.com/linebender/kurbo/issues/335)?

**Comment by nixon-voxell** [comment]
> Would love to see polygons too, but I guess this should be handled upstream ([linebender/kurbo#335](https://github.com/linebender/kurbo/issues/335))?

@musjj It's actually doable using the Bezier Curve, it supports, line, quadratic curve, and cubic curve. Right now it's just that it may be a little tedious to do it, and might need a better interface.

**Comment by nixon-voxell** [comment]
A huge chunk of this issue is blocked by #39 and #30

**Comment by aymey** [comment]
should this be moved to [bevy_vello_graphics](https://github.com/voxell-tech/bevy_vello_graphics)?

**Comment by nixon-voxell** [comment]
@aymey not really, at least not all of them, some of them are not related to vector graphics. I think we will move the individual issues into their respective repo and keep this issue here to track all of them.
